server/nodejs/resources/js/index.js

```diff
@@ -0,0 +1,199 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const surveyData = [
+        {
+            survey_id: 1,
+            survey_title: 'Sample Survey 1',
+            status: 'unpublished',
+            program_id: 101,
+            period_start: '2024-11-01T08:00:00',
+            period_end: '2024-11-15T17:00:00',
+            respondents: { current: 0, total: 50 }
```

Comment:
This needs to be changed as per our previous conversation we agreed upon following this format:

<img width="399" alt="image" src="https://github.com/user-attachments/assets/f3d42e36-f281-4f75-8371-ee4abbb69d2d">